db/src/index.ts

```diff
@@ -55,3 +65,26 @@ export async function update<T extends Serializable = any>(
 export async function find(query: Q.Query): Promise<Document[]> {
   return await db.find(query);
 }
+
+/**
+ * Polls on updates to specified keys since specified versions.
+ */
+export async function poll(keysToVersions: KeyedVersions): Promise<KeyedPatches> {
+  return await db.poll(keysToVersions);
+}
+poll.__shiftjs__ = 'exposed';
+
+/**
+ * Gets a single document by key including its version.
+ */
+export async function getVersioned<T extends Serializable = any>(key: string): Promise<Versioned<T> | undefined> {
+  const doc = await db.getWithMeta<T>(key);
+  if (doc === undefined || doc.value === undefined) {
+    return undefined;
+  }
+  return {
+    version: doc.version,
+    value: doc.value,
+  };
+}
+getVersioned.__shiftjs__ = 'exposed';
```

Comment:
Note that format changed in https://github.com/binaris/shiftjs/pull/94/files#diff-ba07279b055fab1a8980733eebc00d7aR39.